apps/bare-expo/ios/Podfile.lock

```diff
@@ -1993,7 +1993,7 @@ SPEC CHECKSUMS:
   ExpoDevice: 30d5e44aff42048ab016fa0c37a51fdb6b1f8608
   ExpoDocumentPicker: a1a38f72993a4f0622d0c918d520b4ff640ba1cb
   ExpoFileSystem: 1bc21c70ab0bf505cd6bef97c5b008618bfb72f2
-  ExpoGL: 6037e72282fce9746e973c129628cff7f2ee44a6
+  ExpoGL: e452774bffdc758a6fe2dbff589234fa4026a4c3
```

Comment:
Wondering why `ExpoGL` and `expo-dev-launcher` have changed here 🤔

Reply:
I think because Kudo bumped the c++ version [here](https://github.com/expo/expo/pull/25548)